blockchain/message_pool/src/msgpool/mod.rs

```diff
@@ -398,6 +399,149 @@ pub mod tests {
         signed
     }
 
+    #[tokio::test]
+    async fn test_per_actor_limit() {
+        let keystore = KeyStore::new(KeyStoreConfig::Memory).unwrap();
+        let mut wallet = Wallet::new(keystore);
+        let sender = wallet.generate_addr(SignatureType::Secp256k1).unwrap();
+        let target = wallet.generate_addr(SignatureType::Secp256k1).unwrap();
+        let tma = TestApi::default();
+        tma.set_state_sequence(&sender, 0);
+
+        let (tx, _rx) = flume::bounded(50);
+        let mut services = JoinSet::new();
+        let mpool = MessagePool::new(
+            tma,
+            "mptest".to_string(),
+            tx,
+            Default::default(),
+            Arc::default(),
+            &mut services,
+        )
+        .unwrap();
+        let mut smsg_vec = Vec::new();
+        for i in 0..1001 {
```

Comment:
Could we use the `MAX_ACTOR_PENDING_MESSAGES` instead in order to have a single place of truth for this value?